### PR TITLE
FAT-5462/C375288

### PR DESCRIPTION
**Pull request by MaksimTerekhov**
https://foliotest.testrail.io/index.php?/cases/view/375288

https://issues.folio.org/browse/FAT-5462

https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2353/allure/

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

**Review by re-jas — APPROVED** [review]

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(inventoryEntity.item.barcode);
+    FileManager.deleteFile(`cypress/fixtures/${validHoldingUUIDsFileName}`);
+    FileManager.deleteFileFromDownloadsByMask(validHoldingUUIDsFileName, `*${changedRecordsFileName}`, `*${matchedRecordsFileNameValid}`, previewOfProposedChangesFileName.first, previewOfProposedChangesFileName.second, updatedRecordsFileName);
+  });
+
+  it('C375288 Verify generated Logs files for Items In app -- only valid Holdings UUIDs (firebird)', { tags: [testTypes.criticalPath, devTeams.firebird] }, () => {
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js:86]: isn't this a smoke test?

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            inventoryEntity.holdingId = holdings[0].id;
+            inventoryEntity.locations.permanent.id = holdings[0].permanentLocationId;
+            FileManager.createFile(`cypress/fixtures/${validHoldingUUIDsFileName}`, inventoryEntity.holdingId);
+
+            cy.getLocations({ limit: 1, query: `id="${inventoryEntity.locations.permanent.id}"` })
+              .then(loc => { inventoryEntity.locations.permanent.name = loc.name; });
+            cy.getItems({ query: `"barcode"=="${inventoryEntity.item.barcode}"` })
+              .then(inventoryItem => { inventoryEntity.item.id = inventoryItem.id; });
+            cy.getItems({ query: `"barcode"=="secondBarcode_${inventoryEntity.item.barcode}"` })
+              .then(inventoryItem => { inventoryEntity.item.id2 = inventoryItem.id; console.log(inventoryEntity); console.log(inventoryItem); });
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js:74]: i think we don't need console.log here

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js]: thx fixed

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js (hunk 2 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-holdings-valid-uuids.cy.js]: fixed

**Pull request merged**